orthoinference/src/main/java/org/reactome/orthoinference/InferEvents.java

```diff
@@ -0,0 +1,307 @@
+import org.gk.schema.GKSchemaAttribute;
+import org.gk.schema.SchemaClass;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.reactome.release.common.database.InstanceEditUtils;
+
+/**
+ *
+ * @author jcook
+ * 
+ * The Java version of infer_events.pl -- The gist of this module is that it looks at all existing Human ReactionlikeEvent (RlE) instances (mostly Reactions and BlackBoxEvents) in the Test_Reactome database,
+ * and attempts to computationally infer them in each of Reactome's model organisms. Each RlE is broken down into its primary components (input, output, catalyst, and regulator), which are themselves broken
+ * into their PhysicalEntity subunits. The homology data used for the inference process currently comes from Ensembl Compara and is generated during the 'Orthopairs' step of the Reactome release process.
+ * After all inference attempts for each RlE has been completed in an organism, the pathways that contain the reactions are filled with these newly inferred ones. 
+ * 
+ *
+ */
+
+// To Reviewers: I recommend following the code sequentially, rather then looking at each class separately.
+public class InferEvents
+{
+	static MySQLAdaptor dbAdaptor = null;
+	private static GKInstance instanceEdit;
+	private static GKInstance speciesInst;
+	private static HashMap<GKInstance,GKInstance> manualEventToNonHumanSource = new HashMap<GKInstance,GKInstance>();
+	private static ArrayList<GKInstance> manualHumanEvents = new ArrayList<GKInstance>();
+
+	@SuppressWarnings("unchecked")
+	public static void eventInferrer(Properties props, String pathToConfig, String pathToSpeciesConfig) throws Exception
+	{
+		// Set up DB adaptor using config.properties file
+		String speciesToInferFromLong = "Homo sapiens";
+		String username = props.getProperty("username");
+		String password = props.getProperty("password");
+		String database = props.getProperty("database");
+		String host = props.getProperty("host");
+		int port = Integer.valueOf(props.getProperty("port"));
+		
+		dbAdaptor = new MySQLAdaptor(host, database, username, password, port);
+		// Statically store the adaptor variable in each class
+		InferReaction.setAdaptor(dbAdaptor);
+		SkipTests.setAdaptor(dbAdaptor);
+		GenerateInstance.setAdaptor(dbAdaptor);
+		OrthologousEntity.setAdaptor(dbAdaptor);
+		InferEWAS.setAdaptor(dbAdaptor);
+		UpdateHumanEvents.setAdaptor(dbAdaptor);
+		
+		SkipTests.getSkipList("normal_event_skip_list.txt");
+		
+		// For now an array of species names are used. I will likely change it so that the wrapper calls each organism individually during release.
+		List<String> speciesList = new ArrayList<String>(Arrays.asList("pfal", "spom", "scer", "ddis", "cele", "sscr", "btau", "cfam", "mmus", "rnor", "ggal", "tgut", "xtro", "drer", "dmel", "atha", "osat"));
+
+		JSONParser parser = new JSONParser();
+		Object obj = parser.parse(new FileReader(pathToSpeciesConfig));
```

Comment:
Not a priority before release, but it might be cleaner to create a Java class to represent a speciesConfig that handles all the JSON parsing and provides getters that InferEvents can call for the info it needs.